Assets/HoloToolkit-Tests/Utilities/Scripts/Extensions/Editor/GameObjectExtensionsTests.cs

```diff
@@ -0,0 +1,209 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.Tests
```

Comment:
They're technically integration tests since they interact with the unity engine and instantiate several game objects in the tests.